README.md

```diff
@@ -145,3 +145,13 @@ You can access emails in MailHog's Web UI at [http://localhost:8075](http://loca
 
 - [Documentation](https://docs.camunda.org/)
 - [GitHub](https://github.com/camunda/camunda-bpm-platform)
+
+# Troubleshooting
+## Running on arm64 based hardware
+When using arm64-based hardware like a M1 or M2 Mac the Keycloak container might not start because Bitnami only
+provides amd64-based images. You can build and tag an arm-based image locally using following command. After building
+and tagging the image you can start the environment as described in [Using docker-compose](#using-docker-compose).
+
+```
+$ DOCKER_BUILDKIT=0 docker build -t bitnami/keycloak:19.0.3 "https://github.com/camunda/camunda-platform.git#main:.keycloak/"
```

Comment:
❓ what's the reason to explicitly disable buildkit here? it works fine for me with buildkit as well

Reply:
ah, got it. Thanks!